driver/src/price_finding/optimization_price_finder.rs

```diff
@@ -210,7 +211,22 @@ impl PriceFinding for OptimisationPriceFinder {
         &self,
         orders: &[models::Order],
         state: &models::AccountState,
+        time_limit: Duration,
     ) -> Result<models::Solution> {
+        const MINIMUM_SOLVER_TIME_LIMIT: Duration = Duration::from_secs(20);
+
+        // NOTE: A bear mininum amount of time the solver needs for calculating
+        //   a solution. If the time limit is smaller than this minimum, don't
+        //   start the solver at all. Note that this is not considered an error,
+        //   and the trivial solution is returned.
+        if time_limit < MINIMUM_SOLVER_TIME_LIMIT {
+            info!(
+                "Skipping solver run as time limit {} is too short",
+                time_limit.as_secs_f64()
+            );
+            return Ok(models::Solution::trivial(0));
```

Comment:
Again, I think this should be considered an error as we don't process the auction and it should count against our goal of 99% success rate. Not sure if it has to be a hard error, but maybe we can even make our restart scripts smart enough and only restart at the beginning of a batch.

A "real" trivial solution is a success case.

Reply:
Sounds fair.